forage-android/src/main/java/com/joinforage/forage/android/core/telemetry/Metrics.kt

```diff
@@ -32,7 +32,8 @@ internal enum class UnknownForageErrorCode(val value: String) {
 
 internal enum class UserAction(val value: String) {
     BALANCE("balance"),
-    CAPTURE("capture");
+    CAPTURE("capture"),
+    COLLECT_PIN("collect_pin");
```

Comment:
```suggestion
    PIN_DEFER_CAPTURE("pin_defer_capture");
```
How would we feel about `"pin_defer_capture"` to align with [the Forage JS PR](https://github.com/teamforage/fe-monorepo/pull/20#discussion_r1427225857)?

Reply:
We decided on "defer_capture"